src/screens/add-education/add-education.tsx

```diff
@@ -54,18 +53,18 @@ const AddEducationScreen: React.FC<AddEducationRouteProps> = () => {
   };
 
   const initialValues: IEducation = useMemo(() => {
-    if (_education) {
+    if (education) {
       return {
-        id: _education.id,
-        university: _education.university,
-        specialization: _education.specialization,
-        degree: _education.degree,
-        startDate: _education.startDate,
-        endDate: _education.endDate ? _education.endDate : undefined,
+        id: education.id,
+        university: education.university,
+        specialization: education.specialization,
+        degree: education.degree,
+        startDate: education.startDate,
+        endDate: education.endDate ? education.endDate : undefined,
```

Comment:
```suggestion
        endDate: education.endDate ?? undefined,
```
Just a suggestion, no need to fix it